appinventor/blocklyeditor/src/workspace_svg.js

```diff
@@ -260,12 +260,12 @@ Blockly.WorkspaceSvg.prototype.render = function(blocks) {
     if (Blockly.Instrument.useRenderDown) {
       for (var t = 0, topBlock; topBlock = topBlocks[t]; t++) {
         Blockly.Instrument.timer(
-          function () {
-            topBlock.renderDown();
-          },
-          function (result, timeDiffInner) {
-            Blockly.Instrument.stats.renderDownTime += timeDiffInner;
-          }
+            function () {
```

Comment:
It seems like all of the changes in this file with the exception of the addition of centerOnBlock are whitespace changes caused by the IDE. Can you revert any whitespace-only changes?